src/operations/explore_operation.cpp

```diff
@@ -9,12 +9,20 @@
 #include "util.h"
 
 ExploreOperation::ExploreOperation(const std::vector<geometry_msgs::Point>& path)
-    : MoveOperation(OperationIdentifier::EXPLORE, path, 500, 3, 3),
+    : MoveOperation(OperationIdentifier::EXPLORE, path, 500, 0.5, 1),
```

Comment:
I see a lot of different values for the threshold parameters. Could be nice to have comments saying why we choose what?

Reply:
Added an explanation and updated values to all be in [m] (or [m/s])